src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js

```diff
@@ -0,0 +1,59 @@
+const { getDrupalValue } = require('./helpers');
+
+function createMetaTag(type, key, value) {
+  return {
+    __typename: type,
+    key,
+    value,
+  };
+}
+
+const transform = ({
+  title,
+  path,
+  moderationState,
+  metatag: { value: metaTags },
+  fieldNicknameForThisFacility,
+}) => ({
+  entity: {
+    entityType: 'node',
+    entityBundle: 'health_care_region_page',
+    entityPublished: getDrupalValue(moderationState) === 'published',
+    entityLabel: getDrupalValue(title),
+    title: getDrupalValue(title),
+    entityUrl: {
+      path: path[0].alias.replace(/\\/g, ''),
+    },
+    fieldNicknameForThisFacility: getDrupalValue(fieldNicknameForThisFacility),
+    entityMetatags: [
```

Comment:
If we run into this one more time, it might be worth making a function to do all this based on the tags available.

Reply:
Probably a good idea.